pages/treeherder.py

```diff
@@ -70,6 +71,9 @@ def datestamp(self):
         def jobs(self):
             return [self.Job(self.page, root=el) for el in self.find_elements(self._jobs_locator)]
 
+        def pin_all_jobs(self):
+            self.selenium.find_element(*self._pin_all_jobs_locator).click()
```

Comment:
This will find the element from the document (the first one in the page) instead of from the context of the page region. Change it to `self.find_element` and it will find from the root element.